features/guidance/turn.feature

```diff
@@ -1156,12 +1156,12 @@ Feature: Simple Turns
                   |   |
             f._   |   |
                 ' g---h.
-                  |   |  '.
-                  |   |     i
+                  |   |  '-i
+                  |   |
             a_    |   |
                ''.b---c
-                  |   |'d._
-                  |   |    'e
+                  |   |' d._
+                  |   |     'e
                   j   k
```

Comment:
I had the choice to try and hack something into the engine to possibly detect other cases like. Since the original situation works just fine (see screenshot), I opted to reduce the severity of this turn to ensure that I don't spent ages on unnecessary optimisation.
<img width="1163" alt="screen shot 2017-01-26 at 16 41 14" src="https://cloud.githubusercontent.com/assets/12932279/22338038/af0cf9d8-e3e6-11e6-86ea-9bfd44ab416c.png">